react/src/lib/components/SubsurfaceViewer/storybook/SubsurfaceViewer.stories.jsx

```diff
@@ -574,3 +604,40 @@ export const MapInContainer = (args) => {
 MapInContainer.args = {
     ...exampleData[0],
 };
+
+export const Matrix = EditDataTemplate.bind({});
+Matrix.args = {
```

Comment:
Suggest a more descriptive name, eg. "ViewMatrix".

Reply:
Used "new" instead of serializtaion and chenaged the storys name to "ViewMatrix"
I think it is a very good idea to rewrite how the view are set up using a callback (or extern function) as you mentioned. For now I would prefer to get this PR in first and then possibly start a rewrite (which may be some work i suspect).

Reply:
I think it's confusing that we have another story named "Multi View", which btw. doesn't work properly. Perhaps we should remove it or give this story a more specific name, like "ViewMatrixMargin"?